src/components/common/header/PopUp.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+export const PopUp =({popupdisplay, popupFn})=>{ 
+    
+    const handleSubmit =(e)=>{
+       popupFn("popupdisplayHIDE");
```

Comment:
This is not being used, so it can be removed.

Reply:
👍